<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inject_pass">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Fira Code&quot;;</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>258</width>
      <height>312</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Fira Code&quot;;</string>
       </property>
       <property name="text">
        <string>Site web</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_url">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Fira Code&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Generate">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Fira Code&quot;;
color : rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Fira Code&quot;;</string>
       </property>
       <property name="text">
        <string>Mot de passe généré</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="input_pass">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Fira Code&quot;;
text-align: center;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="show_pass">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Fira Code&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_manual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
font: 16pt &quot;Fira Code&quot;;
color: white;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>168</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="edit_row">
       <property name="text">
        <string>Edit row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_row">
       <property name="text">
        <string>Delete row</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionClear_db"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionClear_db">
   <property name="text">
    <string>Clear db</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
